kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/assertions/nondeterministic/eventually.kt

```diff
@@ -156,7 +161,7 @@ class EventuallyConfigurationBuilder {
    var intervalFn: DurationFn? = EventuallyConfigurationDefaults.intervalFn
 
    /**
-    * The maximum number of invocations regardless of durations. By default, this is set to max retries.
+    * The maximum number of invocations regardless of durations. By default, this is set to max retries. And Must be greater than or equal to 0.
```

Comment:
```suggestion
    * The maximum number of invocations regardless of durations. By default, this is set to max retries. Must be non-negative.
```

nitpick: match the KDoc of `duration`.